src/main/java/mate/academy/spring/service/mapper/CinemaHallDtoMapper.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.spring.service.mapper;
+
+import mate.academy.spring.model.CinemaHall;
+import mate.academy.spring.model.dto.CinemaHallRequestDto;
+import mate.academy.spring.model.dto.CinemaHallResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CinemaHallDtoMapper {
+    public CinemaHallResponseDto parse(CinemaHall cinemaHall) {
```

Comment:
```suggestion
    public CinemaHallResponseDto toDto(CinemaHall cinemaHall) {
```
same in other 2 mappers